cmd/ttn-lw-cli/commands/applications_packages.go

```diff
@@ -259,6 +304,146 @@ var (
 				return err
 			}
 
+			return nil
+		},
+	}
+	applicationsPackagesDefaultAssociationsCommand = &cobra.Command{
+		Use:     "default-associations",
+		Aliases: []string{"def-assoc", "def-assocs"},
+		Short:   "Application packages default associations commands",
+	}
+	applicationsPackageDefaultAssociationGetCommand = &cobra.Command{
+		Use:     "get [application-id] [f-port]",
+		Aliases: []string{"info-association"},
```

Comment:
This doesn't make a lot of sense to me. `info` maybe, but not `info-association`